src/main/java/org/sonarlint/intellij/core/SonarLintEngineFactory.java

```diff
@@ -143,7 +143,7 @@ private static Map<String, String> prepareExtraProps() {
 
   private static void addOmnisharpServerPaths(SonarLintPlugin plugin, Map<String, String> extraProps) {
     extraProps.put("sonar.cs.internal.omnisharpMonoLocation", plugin.getPath().resolve("omnisharp/mono").toString());
-    extraProps.put("sonar.cs.internal.omnisharpWinLocation", plugin.getPath().resolve("omnisharp/win").toString());
+    extraProps.put("sonar.cs.internal.omnisharpWinLocation", plugin.getPath().resolve("omnisharp/net472").toString());
```

Comment:
Interesting, what was the impact of this?

Reply:
I just changed the folder name where the omnisharp files are deployed to. Before it was `win`, now I keep the name of the artifact classifier, which is `net472`